application/src/main/java/uk/gov/hmcts/ccd/definition/store/ApplicationConfiguration.java

```diff
@@ -40,4 +44,12 @@ public Executor asyncExecutor(@Value("${async.executor.core.pool.size}") Integer
     public CacheManagerCustomizer<CaffeineCacheManager> cacheManagerCustomizer() {
         return cacheManager -> cacheManager.setAllowNullValues(false);
     }
+
+    @Bean
+    public PlatformTransactionManager transactionManager(final EntityManagerFactory emf,@Value("${ccd.tx-timeouts.default}") String timeout) {
```

Comment:
Could we move this 'timeout' from the parameter to inside the method?

Reply:
this is not a common attribute to the hole class, timeout is not an attribute .  It is a parameter only used for Bean creation.